testbed/coav-sim.sh

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env bash
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SCRIPT_DIR=$(dirname $(realpath ${BASH_SOURCE[0]}))
+
+# Supported Autopilots
+AP_PX4=0
+AP_APM=1
+
+# ArduCopter Variables
+# APM_DIR will be added to PATH if set
+APM_CMD="arducopter"
+APM_TCP_PORT_1=5760
+APM_TCP_PORT_2=5762
+
+# PX4 Variables
+PX4_DIR=${PX4_DIR:-"~/px4/Firmware"}
+PX4_CMD="make posix_sitl_default jmavsim"
+PX4_UDP_PORT_1=14550
+PX4_UDP_PORT_2=14540
+
+# Simulation Parameters
+WORLD=${WORLD:-"simple_obstacle.sdf"}
+
+# Collision Avoidance and Gazebo variables
+GZSITL_UDP_PORT=15556
+COAV_GCS_UDP_PORT=15557
+
+# Set autopilot
+AUTOPILOT=${AUTOPILOT:-AP_PX4}
+
+run_autopilot () {
+        # Run SITL Simulator
+    if (("$AUTOPILOT" == "$AP_PX4")); then
```

Comment:
Here I think that the magic between jmavsim + gazebo plugin isn't work. It looks like both environment are trying to set stuff on the drone (like the home position), it doesnt arm or takeoff but itself and if I open a GCS complex mission is loaded, I can arm and takeoff via GCS and the drone does crazy things on jmavsim and remain dead on gzclient.